tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,93 @@
+"""Module2/task1: Includes task of creating sandwich calculator based on user's input."""
+import pyinputplus as pyip
+
+
+# Vars
+BREAD_TYPES_AND_PRICES = {'wheat': 4, 'white': 2, 'sourdough': 3}
+PROTEIN_TYPES_AND_PRICES = {'chicken': 6, 'turkey': 7, 'ham': 7, 'tofu': 4.85}
+CHEESE_TYPES_AND_PRICES = {'cheddar': 5, 'Swiss': 9, 'mozzarella': 7.5}
+SAUCE_TYPES_AND_PRICES = {'mayo': 2.5, 'mustard': 2, 'lettuce': 1, 'tomato': 0.75}
+
+
+def select_bread() -> dict:
+    """Select type of a bread.
+
+    :return {bread_type, bread_price}
+    """
+    print("What type of bread do you want?")
+    declared_bread = pyip.inputMenu([*BREAD_TYPES_AND_PRICES.keys()])
+    return {bread: price for bread, price in
```

Comment:
Isn't it the same as `return {declared_bread: BREAD_TYPES_AND_PRICES[declared_bread]}`?

IMO it is much more readable than using dict comprehension.

Reply:
Indeed, a lot more readable 🗡️